frontend/packages/shared/src/contexts/ServicesContext.tsx

```diff
@@ -57,6 +57,7 @@ const handleError = (
 
   if (unAuthorizedErrorCode) {
     renderToast(errorCode || 'Unauthorized', {
+      onClose: () => logout().then(() => window.location.assign(userLogoutAfterPath())),
```

Comment:
We can consider to change this function to a new const so we can call the same function in line 60 and 64, with a simple descriptive name like `LogOutUser()` or something? 😊

Reply:
fixed😊